gateway/tests/resources/fake_media/test_user/results/8317718f-5c0d-4fb6-9947-72e480b8a348.json

```diff
@@ -0,0 +1 @@
+{"ultimate": 42}
```

Comment:
@paaragon similar to the previous one, can this file be deleted? I don't see is needed in any test but just to confirm.

Reply:
Oh yes... is not needed in this branch but in [feat/retrieve-job-results-from-result-store](https://github.com/Qiskit/qiskit-serverless/pull/1575) I'm removing it from here